### PR TITLE
[SIM_FLUTTER-81] [Integrate] Implement Admin Update User

### PR DESCRIPTION
**Pull request by egiegarciano14**
### Links
- [Backlog](https://framgiaph.backlog.com/view/SIM_FLUTTER-81)
- [Slack](https://sun-philippine.slack.com/archives/C059L7XAXFB/p1689665240275289)

### Definition of done
- [x] Admin can update other users profile info
- [x] Admin can update other users password

### Notes
- Login as admin

### Recordings
https://github.com/framgia/sph-flutter/assets/79126024/34085857-479d-410f-9ce0-f1bfb9f4299e

### CODE REVIEW

**Review by Dbulawan13 — DISMISSED** [review]
Approve PR #73 

Working:
- Able to update all Users Information
- Able to update Password
- Validations still work properly

https://github.com/framgia/sph-flutter/assets/86758474/5ea64088-3272-48ec-87bb-703105ded085

https://github.com/framgia/sph-flutter/assets/86758474/b3860716-08f5-4f0c-a56d-f34713af15e8

Testing Hours: 0.16
Comment: 0
Iteration: 0

File: frontend/lib/src/features/user_profile/profile_info.dart [hunk 1]
```diff
@@ -52,136 +71,158 @@ class ProfileInfo extends StatelessWidget {
...
-                    formKey.currentState?.fields['address']
-                        ?.validate(focusOnInvalid: false);
-
-                    controller.setFirstName =
-                        formKey.currentState?.fields['first_name']?.value ?? '';
-                    controller.setMiddleName =
-                        formKey.currentState?.fields['middle_name']?.value ??
+              Obx(
+                () {
+                  if (controller.isLoading) {
```
Comment (jstephend-sun) [inline frontend/lib/src/features/user_profile/profile_info.dart]: remove "is"

rename to a more descriptive name
relate it with the submission of data

File: frontend/lib/src/navigators/users_list_screen_navigator.dart [hunk 2]
```diff
@@ -20,6 +26,32 @@ class UserListScreenNavigator extends StatelessWidget {
...
+                );
+              },
+            );
+            break;
+          case '/profileChangePassword':
+            final args = settings.arguments as ProfileScreenArguments;
+            page = ProfileChangePassword(
+              profileUserId: args.userId,
+              backIconOnTap: () {
+                settingsAppNav.currentState?.pop();
```
Comment (jstephend-sun) [inline frontend/lib/src/navigators/users_list_screen_navigator.dart]: use of maybePop for safety

File: frontend/lib/src/navigators/users_list_screen_navigator.dart [hunk 3]
```diff
@@ -20,6 +26,32 @@ class UserListScreenNavigator extends StatelessWidget {
...
             page = const AdminUserListPage();
             break;
+          case '/profileInfo':
+            final args = settings.arguments as ProfileScreenArguments;
+            page = ProfileInfo(
+              profileUserId: args.userId,
+              withBackIcon: true,
+              backIconOnTap: () async {
+                await controller.getUsers();
+                settingsAppNav.currentState?.pop();
```
Comment (jstephend-sun) [inline frontend/lib/src/navigators/users_list_screen_navigator.dart]: maybepop

**Review by jstephend-sun — CHANGES_REQUESTED** [review]
Requested Changes #73

review time: 10mins
comments: 2

**Review by jstephend-sun — APPROVED** [review]
Approved #73

review time: 10mins
comments: 2
iteration: 1st review

**Pull request merged**